assemblies/assembly-install-rhdh-ocp.adoc

```diff
@@ -1,19 +1,30 @@
 [id='assembly-install-rhdh-ocp']
 
-= Deploying {product} on {ocp-short}
+= Installing {product} on {ocp-short}
+:context: install-rhdh-ocp
 
 You can install {product} on {ocp-short} using one of the following methods:
 
-* The Helm chart
-* The {product} Operator
+* *The {product} Operator*
+** Ready for immediate use in {ocp-short} after an administrator installs it with OperatorHub
+** Can be configured for use in Kubernetes
+** Updates automatically with Operator Lifecycle Management (OLM)
 
-// Helm chart method
-include::modules/installation/proc-install-rhdh-ocp-helm.adoc[leveloffset=+1]
-include::modules/getting-started/proc-add-custom-app-file-openshift-helm.adoc[leveloffset=+2]
-include::modules/installation/proc-install-rhdh-airgapped-environment-ocp-helm.adoc[leveloffset=+2]
+* *The {product} Helm chart*
+** Ready for immediate use in both {ocp-short} and Kubernetes
+** Requires manual installation and management
+
+Use the installation method that best meets your needs and preferences.
```

Comment:
```suggestion
Use the installation method that best meets your needs, cluster type, and preferences.
```

Reply:
Specific to OCP, so we might omit cluster type. Or should we consider variants in the OpenShift family?

Reply:
AFAIU the docs should just talk about "OpenShift" ... the various flavours of OCP vs. OSD vs. OS Local are all the same (basically).

Reply:
I agree with @nickboldt, it shouldn't matter unless they are using an air-gapped cluster.